### DISCUSSION TITLE
Improve missing master key error

### DISCUSSION THREAD

**Discussion by CaroFG** [meilisearch/product#565] [Feedback & Feature Proposal]
Related to #536 

When launching Meilisearch with no master key but the environment set to production, Meilisearch throws the following error:

`Error: In production mode, the environment variable MEILI_MASTER_KEY is mandatory`

The message only mentions the env var, but not the CLI option `--master-key`

**Comment by gmourier** [comment]
Thanks @CaroFG 

Would it be better with `Error: In production mode, a master key must secure the instance access. It can be specified via the MEILI_MASTER_KEY environment variable or the --master-key launch option` ?

**Reply by gmourier** [reply]
@dichotommy If you have a better idea to rephrase my suggestion, it would be really appreciated 🙇‍♂️

**Reply by CaroFG** [reply]
It's better, indeed :) Thank you

**Reply by dichotommy** [reply]
`Error: In production mode, you must provide a master key to secure your instance. It can be specified via the MEILI_MASTER_KEY environment variable or the --master-key launch option.`

**Reply by curquiza** [reply]
Sorry, just a detail, but what about the config file? 🙈

**Reply by gmourier** [reply]
I thought about it, and I said to myself that it is not so important to talk about the config file since it maps the env var, as the options flag by essence.

It seems a bit redundant to mention the config file here, but why not?

If a proposed phrasing mentions it, we could take that one.

**Comment by gmourier** [comment]
Specified here https://github.com/meilisearch/specifications/pull/209 (Suggested the message from @dichotommy for the related error)

**Comment by gmourier** [comment]
Hey everyone! 👋

I'm locking that discussion since it will be released with `v1`

Feel free to open a new discussion to share feedback or ask for a feature request.

Thank you 🙇‍♂️
